src/dumpling-service/dumpling.csharp-client/.gitignore

```diff
@@ -0,0 +1,185 @@
+# Ref: https://gist.github.com/kmorcinek/2710267
```

Comment:
Is the default .gitignore at the base of the repo not sufficient?  If not we should update this rather than having multiple .gitignore files scattered in the repo.